4/완주하지못한선수-2.js

```diff
@@ -0,0 +1,15 @@
+function solution(participant, completion) {
+  const runners = {};
+
+  for (let p of participant) {
+    runners[p] = runners[p] ? runners[p] + 1 : 1;
+  }
+
+  for (let c of completion) {
+    runners[c] -= 1;
+  }
+
+  for (let r in runners) {
+    if (runners[r] === 1) return r;
```

Comment:
```suggestion
    if (runners[r]) return r;
```
'=== 1' 생략 가능합니당
조건문이 1 이상인 경우 모두 참으로 간주합니다.